Modules/offline_awareness/__init__.py

```diff
@@ -0,0 +1,43 @@
+"""
+__init__.py - Offline awareness base implementation
+
+Provides a ABC and an interface for offline-aware components.
+Specifically, this module provides the :class:`OfflineAwareABC` base class, which provides an
+interface and signalling primitive for offline-aware classes.
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+
+from ._offline_aware_abc import OfflineAwareABC
+
+# convenience aliases
```

Comment:
Nitpick: This comment seems a bit too high up from the code it relates to below, it almost looks like a blank section.

Reply:
looks more like a stray comment, will purge

Reply:
resolved in e531764